plugins/modules/gitlab_group_access_token.py

```diff
@@ -185,7 +185,11 @@ def create_access_token(self, group, arguments):
     @param name of the access token
     '''
     def find_access_token(self, group, name):
-        access_tokens = group.access_tokens.list(all=True)
+        def is_revoked(token):
+            if hasattr(token, 'revoked') and token.revoked is not None:
```

Comment:
This could be simplified as:
```suggestion
            if getattr(token, 'revoked', None):
```

Reply:
Thanks a lot for your suggestion, it actually gave me the opportunity to completely remove the internal function and use `getattr()` directly with a default value of `False`. I also added a new test to check that we are indeed backward compatible with older versions of Gitlab.